src/imgui/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # imgui test executable, with full warnings enabled
-add_executable(test_imgui test.cpp)
+add_executable(test_imgui test.cpp FileHandler.hpp)
```

Comment:
Missing the `test.cpp` on the PR

Reply:
you mean a test class for this 😛 ?

Reply:
test.cpp is our main. we have to rename this, btw

Reply:
how can it be missing?

Reply:
it is not, it's there

Reply:
@soaresiv